docs/chapter-5.mdx

```diff
@@ -1,90 +1,205 @@
+
+```ts title="app.component.ts"
+import { Component, OnInit } from "@angular/core";
+import { addCategory } from "./state/actions";
+
+@Component({/* ... */})
+export class AppComponent implements OnInit {
+  ngOnInit() {
+    console.log(addCategory({category:{ name: "Food" }}));
+  }
+}
+```
+
+In the console, we will see the following:
+
+```js
+{category: {name: 'Food', type: '[Category List] Add Category'}}
+```
+
+So essentially, `createAction` provided us with an easy way of creating a single action. `addCategory` in our case is an `ActionCreator`, a function which produces an action object whenever called, and `props` explained what argument that `ActionCreator` function expects.
+
+#### Using `createActionGroup`
+
+As we just saw, creating an action is pretty straightforward.
+
+However, when dealing with multiple actions, this can led to a more verbose code and prone to typos:
+
+```ts title="state/actions.ts"
+export const addCategory = createAction(
+  "[Category List] Add Category",
+  props<{ category: Category }>()
+);
+
+export const addAnotherCategory = createAction(
+  //  👇 Did you notice the typo?
+  "[Catgory List] Add Another Category",
+  props<{ category: Category }>()
+);
+```
+
+With those issues in mind, NgRx published a new function that allows us to create multiple actions at one: the `createActionGroup` function.
+
+Its usage is fairly simple and is based on two parameters:
+
+- The source of the action, in our case `Category List`
+- A serie of properties symbolizing our actions and their parameters
+
+Let's go back to the previous example to see what is changing:
+
+```ts title="state/actions.ts"
+export const categoryActions = createActionGroup({
+  source: 'Category List',
+  events: {
+    'Add Category': props<{ category: Category }>(),
+    'Add Another Category': props<{ category: Category }>(),
+  }
+})
+```
+
+By doing so, all our related actions will be grouped together and we still can create them by calling the `categoryActions` variable that automatically generates you the code to create them:
+
+```ts
+categoryActions.addCategory({ category: 'Food' });
+categoryActions.addAnotherCategory({ category: 'Food' });
+```

```

Comment:
I think we would do better if we explain the magic behind  converting the plain English words into camelCase property names. You can refer to [my article](https://dev.to/this-is-angular/the-new-ngrx-lf1) , the section titled "Action Groups"

Reply:
Thanks for this addition!